jsk_rviz_plugins/src/pie_chart_display.cpp

```diff
@@ -359,6 +363,11 @@ namespace jsk_rviz_plugins
     
   }
   
+  void PieChartDisplay::updateCaption()
+  {
+    caption_ = caption_property_->getStdString();
```

Comment:
Btw, please look at .rviz file in this commit. It uses old format and works ok, so we do not need https://github.com/jsk-ros-pkg/jsk_visualization/pull/636#diff-for-comment-78361717 for backward compatibility